src/main/java/seedu/duke/commands/ListCommand.java

```diff
@@ -27,10 +33,36 @@ public void execute(TextUi ui, ListManager listManager) {
             CategoryList categoryList = (CategoryList) listManager.getList(ListManager.CATEGORY_LIST);
             listCategories(categoryList, ui);
             break;
+        case TAG_RATING:
+            RatingList ratingList = (RatingList) listManager.getList(ListManager.RATING_LIST);
+            listRatings(ratingList, ui);
+            break;
         default:
         }
     }
 
+    private void listRatings(RatingList ratingList, TextUi ui) {
+        ArrayList<Rating> ratings = ratingList.getList();
+        ratings.sort(Comparator.comparing(Rating::getRating));
+        Collections.reverse(ratings);
+        if (information == "") {
```

Comment:
Not sure if this line of code will fail, but I think it will be better to use information.isEmpty() or information.equals("") ?

Reply:
Yeah, didn't notice it. Should have used information.equals("") since information is String. Thanks for spotting :)